docs/source/en/model_doc/seggpt.md

```diff
@@ -26,7 +26,8 @@ The abstract from the paper is the following:
 
 Tips:
 - One can use [`SegGptImageProcessor`] to prepare image input, prompt and mask to the model.
-- It's highly advisable to pass `num_labels` (not considering background) during preprocessing and postprocessing with [`SegGptImageProcessor`] for your use case.
+- When using binary masks or segmentation maps as prompts pass it to [`SegGptImageProcessor`] `segmentation_maps` argument, if you already converted your prompt mask to RGB then pass it to `prompt_masks` argument.
+- It's highly advisable to pass `num_labels` when using `segmetantion_maps` (not considering background) during preprocessing and postprocessing with [`SegGptImageProcessor`] for your use case.
```

Comment:
No, let's not do this. Having two different inputs like this is both error prone and an antipattern. The image processor can selectively process based on the `do_xxx` flags e.g. if the input image was already resized, we wouldn't have a different input e.g. `resized_images` we would still pass in `images` and set `do_resize=False`. In this case, it looks like what we want is `do_convert_rgb`

Reply:
It makes perfect sense! Introduced the `do_convert_rgb` flag and updated the tests accordingly here https://github.com/huggingface/transformers/pull/29550/commits/2185b3e47de9714d6fa5213ddd30d7dc73427a94